elections/steering/README.md

```diff
@@ -1,153 +1,10 @@
-## Kubernetes Elections
+# Kubernetes  Steering Elections
 
-This document is a guide for Election Committee members
-conducting a Kubernetes Steering Committee Election.
+This folder contains information on the Kubernetes Steering elections since 2017.
 
-It is currently a Work In Progress, and contains much that is outdated.
+The current Steering election, including all directions on eligibility, voting, and candidates, can be found here: [2022 Election]
```

Comment:
Should this say the previous Steering election? Or would this be better phrased as "the most recent Steering election" and then it can be updated to point to the 2023 Election kicks off, and rolled forward in that manner? 2022 doesn't really seem like the "current" election

Reply:
Hmmm, I suppose so.  Realistically it won't be updated after every election, but we might as well begin right.